cypress/integration/navigations.ts

```diff
@@ -0,0 +1,58 @@
+import { IFRAME_SELECTOR } from '../support/constants';
+
+describe('navigations', () => {
+  beforeEach(() => {
+    cy.loadPage('/streamed-epub');
+  });
+
+  it('table of content link should update the page content', () => {
+    cy.iframe(IFRAME_SELECTOR)
+      .findByRole('img', {
+        name: "Alice's Adventures in Wonderland, by Lewis Carroll",
+      })
+      .should('exist');
+
+    // Open TOC menu
+    cy.findByRole('button', { name: 'Table of Contents' }).click();
+
+    // Open chapter 1
+    cy.findByRole('menuitem', { name: 'I: Down the Rab­bit-Hole' }).click();
+    cy.wait(100); // wait for the nested iframe to render
+
+    cy.iframe(IFRAME_SELECTOR)
+      .find('.subtitle')
+      .contains('Down the Rab­bit-Hole');
```

Comment:
I would personally add this same selector but asserting it does not exist before you navigate. That way we are sure it's not passing simply because we are in scrolling mode and that subtitle appears further down the page.